Schemas/visinfo.xsd

```diff
@@ -25,7 +25,7 @@
 					</xs:complexType>
 				</xs:element>
 				<!-- ISG Jira issue BCF-17: Add support for text in the viewpoints -->
-				<xs:element name="Bitmap" minOccurs="0" maxOccurs="unbounded">
+				<xs:element name="Bitmaps" minOccurs="0" maxOccurs="unbounded">
```

Comment:
This would rename the individual elements each to `Bitmaps`, but they'd still all appear in a sequence in the parent object. I think it would be better to create a wrapper element `Bitmaps` which holds one or many `Bitmap` elements, like this:

```xml
<Visinfo>
  <Bitmaps>
	  <Bitmap>
		<Reference>123</Reference>
	  </Bitmap>
	  <Bitmap>
		<Reference>456</Reference>
	  </Bitmap>
	  <Bitmap>
		<Reference>789</Reference>
	  </Bitmap>
  </Bitmaps>
</Visinfo>
```

Reply:
Ah, yes. This was my intention, but done wrong.

Reply:
@GeorgDangl Please check again. It looks similar to Lines/Line or ClippingPlanes/ClippingPlane to me (with some exceptions on minOccurs/maxOccurs).